tests/dataset_histograms/histogram_error_estimator_test.py

```diff
@@ -121,6 +139,17 @@ def test_get_ratio_dropped_linf(self, linf_bound, expected):
         self.assertAlmostEqual(estimator.get_ratio_dropped_linf(linf_bound),
                                expected)
 
+    @parameterized.parameters((0, 1), (0.5, 0.89), (1, 0.78), (2, 0.76),
+                              (40, 0))
+    # there 1 is contribution 40 and 10 contribution 1.
```

Comment:
there is 1 contribution of 40 and 10 contributions of 1.

Reply:
thx